services/dashboard/views/apps/streamingapp/streamingapp.html

```diff
@@ -8,13 +8,13 @@
       </h4>
       <h5 ng-bind="app.description"></h5>
     </div>
-    <div class="col-md-2 col-sm-3 col-xs-4">
+    <div class="col-md-2 col-sm-3 col-xs-3">
```

Comment:
"Application clock" is expected to align with the "akka.tcp://...".

In general, i would suggest break "2 processors 3 executors" in next line. As result: "col-xs-4", "col-xs-8", "col-xs-12"